src/components/HelpSection.vue

```diff
@@ -289,6 +261,20 @@ export default {
 }
 </script>
 <style scoped>
+    .help-link {
+        padding-bottom: 8px;
+    }
+    .help-link .help-link-a {
+        color: #ED592F;
+    }
+    .help-links .help-link .help-link-a:hover {
+        font-weight: bold;
+        color:black!important;
```

Comment:
The hover color of other links is not black, but `#363636`. Please change to this color to maintain consistency.